Components/Signup/Signup.tsx

```diff
@@ -0,0 +1,236 @@
+
+import styles from './Signup.module.scss'
+function Signup({ login, setLoginKey }) {
+  const onClose = (e) => {
+    e.preventDefault();
+    setLoginKey(false);
+  };
+
+  if (login === false) {
+    return null;
+  } else {
+    return (
```

Comment:
You can remove lines 9-11 and change to this

```suggestion
    return login && (
```